### PR TITLE
Feature/submodule defi

### PR DESCRIPTION
**Pull request by ruckim**

### CODE REVIEW

File: packages/tatum-core/src/model/response/offchain/Wallet.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+export type Wallet = WalletWithMnemonic | WalletWithAddress
+
+export interface WalletWithMnemonic {
```
Comment (prokopsimek) [inline packages/tatum-core/src/model/response/offchain/Wallet.ts:3]: Should we also move this interface to define library? There is a problem that tatum-ledger also needs these interfaces but we can add dependency to tatum-ledger on tatum-defi.

File: packages/tatum-defi/tsconfig.json [hunk 2]
```diff
@@ -13,8 +13,7 @@
       "node_modules/@types"
     ],
     "types": [
-      "node",
-      "jest"
+      "node"
```
Comment (prokopsimek) [inline packages/tatum-defi/tsconfig.json:16]: revert jest back

**Review by prokopsimek — CHANGES_REQUESTED** [review]
You have to also delete/refactor nft and wallet folders in each chain.

File: packages/tatum-core/src/model/response/offchain/Wallet.ts (hunk 1 above)
Comment (samuelsramko) [reply packages/tatum-core/src/model/response/offchain/Wallet.ts:3]: move to ledger pls and add as dependency to defi

Comment (Hathoriel) [reply packages/tatum-core/src/model/response/offchain/Wallet.ts:3]: @prokopsimek I would add tatum-core as a dep for tatum-ledger, thats ok?

Comment (prokopsimek) [reply packages/tatum-core/src/model/response/offchain/Wallet.ts:3]: @Hathoriel `tatum-core` already is a dependency of `tatum-ledger`.

**Pull request merged**